modules/newhorizons/rendering/renderableplaneprojection.cpp

```diff
@@ -265,8 +265,9 @@ void RenderablePlaneProjection::updatePlane(const Image& img, double currentTime
     if (!_moving) {
         SceneGraphNode* thisNode = OsEng.renderEngine().scene()->sceneGraphNode(_name);
         SceneGraphNode* newParent = OsEng.renderEngine().scene()->sceneGraphNode(_target.node);
-        if (thisNode != nullptr && newParent != nullptr)
-            thisNode->setParent(newParent);
+        if (thisNode != nullptr && newParent != nullptr) {
```

Comment:
Autocasting to `bool` is fine for `ptr` types:

`if (thisNode && newParent) {`